packages/taxdat/R/model_evaluation_helpers.R

```diff
@@ -580,11 +603,11 @@ plot_model_fidelity <- function(data_fidelity,
       ggplot2::geom_point(ggplot2::aes(y = `modeled cases`, x = `actual cases`, col = chains)) +
       ggplot2::geom_abline(intercept = 0, slope = 1) +
       ggplot2::coord_fixed(ratio = 1, xlim = c(1, max(comparison[[1]][,3:4])), ylim = c(1, max(comparison[[1]][,3:4]))) +
-      ggplot2::theme_bw() +
+      ggplot2::themec_bw() +
```

Comment:
this isn't a real function in my ggplot2? do i need to update, or is this a typo, or..?

Reply:
oops this is a typo, i've updated it.